packages/web/__tests__/pages/t/index.js

```diff
@@ -40,4 +40,17 @@ describe('Technology Details Page', () => {
 
 		expect(container).toMatchSnapshot();
 	});
+
+	it('should not render `Custo de implantação` if technology has no cost', () => {
```

Comment:
```suggestion
	it('should not render implementation costs if the technology does not have it', () => {
```
Me corrija se eu estiver errado hahah :sweat_smile:

Reply:
Eu me referi ao parágrafo, mas também serve 😅